README.md

```diff
@@ -86,4 +91,4 @@ Or you create your own; it doesn't strictly have to be a fork.
 
 ### Or Submit a Pull Request
 
-We also accept pull requests on this repository, if they're small!
\ No newline at end of file
+We also accept pull requests on this repository, if they're small!
```

Comment:
Would we accept PRs now? Should we tell people not to submit them?

Reply:
I think it's OK to accept PRs, e.g. if someone doesn't update their project to the new template then they might make changes in this repo to fix bugs. It's not recommended to stay with this repo, but then again we can't stop people!